### ISSUE TITLE
Unknown behaviour of GPModel with shared fits

### ISSUE THREAD

**Issue by taylorbell57** [kevin218/Eureka#258]
What is the behaviour of the GPModel when doing a shared fit? The fact that `GPModel.__init__()` doesn't take in longparamlist, nchan, or paramtitles makes me pretty strongly suspect that we will get unexpected behaviours and/or a full crash if a GP is used in a shared fit. Has this been tested?

**Comment by evamariaa** [comment]
Yes, that will likely be a problem. It was not on top of my priority list to make it work with the shared parameters fit as I think it's computationally not possible to run a fit with shared GP parameters, at least when using george. Thanks for opening this issue (I should've done that a couple of weeks ago when I realised - sorry!).  
I'll fix it after I have included celerite and/or tinygp as a GP package.
